fiftyone/utils/labelstudio.py

```diff
@@ -636,7 +636,11 @@ def export_label_to_label_studio(label, full_result=None):
         a dictionary or a list in Label Studio format
     """
     # TODO model version and model score
-    if _check_type(label, fol.Classification, fol.Classifications):
+    if label is None:
+        result_value = {}
+        ls_type = ""
+        ids = [0]
```

Comment:
which values should ls_type and ids have?

Reply:
```suggestion
        ls_type = None
        ids = []
```

`ids` will need to be an empty list in this case. Since `ids` and `result_value` are empty, `ls_type` will not be used. I would just set it to `ls_type = None`.